Doc updates for README and Getting Started guide (#413)
aws-ia/terraform-aws-eks-blueprints: docs/getting-started.md

# Getting Started

This getting started guide will help you deploy your first EKS environment using EKS Blueprints.

## Prerequisites:

> Ensure that you have installed the following tools in your Mac or Windows Laptop before start working with this module.

Reviewer: ```suggestion
First, ensure that you have installed the following tools locally.
```

✅  AWS Load Balancer Controller  
✅  Cluster Autoscaler  
✅  CoreDNS  
✅  kube-proxy  
✅  Metrics Server  
✅  vpc-cni  

### Clone the repo

```
git clone https://github.com/aws-ia/terraform-aws-eks-blueprints.git
```

### Terraform INIT

CD into the example directory:

```
cd examples/eks-cluster-with-new-vpc/
```

Initialize the working directory with the following:

```
terraform init
```

### Terraform PLAN

Verify the resources that will be created by this execution:

```
terraform plan
```

### Terraform APPLY

We will leverage Terraform's [target](https://learn.hashicorp.com/tutorials/terraform/resource-targeting?in=terraform/cli) functionality to deploy a VPC, an EKS Cluster, and Kubernetes add-ons in separate steps. 

**Deploy the VPC**. This step will take roughly 3 minutes to complete. 

```

> terraform apply -target="module.aws_vpc"

Reviewer 2: 👍🏽
